doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
-   > * Standard CNTT methodology (i.e. normalize interfaces, APIs, capabilities and behaviors at the reference points) applies well to networking, helping mitigate the need to be overly prescriptive about implementation  
-   > * CNTT/OPNFV MUST provide a functional networking solution for the RI, and in support of RC  
-   > * CNTT/OPNFV potentially does NOT have to provide a production networking solution for RI  
-   > * CNTT/OPNFV is NOT planning to provide a production compute solution for RI
+•	ETSI NFV does not have a separation between HW Infrastructure Management and SW Virtualisation Management and by that the cardinality of having multiple CaaS and IaaS layers on top of a shared HW Infrastructure Layer cannot be expressed
 
+•	ETSI NFV lacks a description of the reference points in between the SW Virtualisation Layers and the HW Infrastructure Layer denoted as Vl-Ha and by that cannot express Packet Flows, Control/Status Interfaces and Management Interfaces between them
 
-<a name="5.1.1"></a>
-### 5.1.1 Executive Summary
+•	ETSI NFV model does not explicitly call out an HW Abstraction layer on top of the HW resources and by that many SW Virtualisation Layers act upon the physical HW components in a non-portable way that do not allow an HW Infrastructure to compose abstracted HW resources from pools of individual HW components
+
+•	ETSI NFV lack of spelled out HW Abstraction does not enable a clear way to abstract and describe the technical possibilities that HW layer networking separation, encapsulation, acceleration, etc. can be implemented either in a HW Infrastructure Managed SmartNIC on the Server units or in the shared underlay switching fabric i.e. on the physical Switching Units 
+
+•	ETSI NFV model does not include any reference to SDN (Software Define Network) controller(s) and relevant integration points into NFVI and MANO 
+
+•	ETSI NFV model does not have a way to enable programmable forwarding planes in the HW layer controlled from higher layers of virtualization managers, orchestrators or Network Functions
+
+<p align="center"><img src="../tech/figures/ETSI.png"><b> Figure 5-1-1-1: ETSI model and Networking.</b></p>
+
+Note: Other areas of concern may be identified and addressed during the later stages of work on the RM/RA contents for Networking.
+
+<a name="5.1.1.2"></a>
+#### 5.1.1.2 Specific topics to be included in the Networking Scope
+To address the concern areas detailed above the specific topics listed below will be covered under Networking. The list below does not pretend to be complete. It will be a subject to modifications as a result of the community discussion.
+
+•	Network connectivity for L3 tenants, GWs, SDS, etc
+
+•	Spine-Leaf Architecture ( 3 Tier or 5 Tier CLOS architecture )
+
+•	Automation of the programmable newtorking fabric
+
+•	Variety of Underlay protocols that can be used as a transport layer within the physical fabric
+
+•	Variety of Overlay protocols
```

Comment:
Need to consider how the encapsulation protocol impacts the workload (it shouldn't). Assuming no impact, encap becomes an implementation detail (i.e. out of scope).

Reply:
@markshostak - I agree with you that this should not impact the workload, but, since this chapter is specifically about the networking, it is probably good to specify.

Reply:
@markshostak @sukhdevkapur good comments.  They should not.  An this should be specifically said, hence it is in the scope.

Reply:
@walterkozlowski @sukhdevkapur I'm ok to reference encap protocol selection should (i.e. shall) have no impact on the reference point seen by the workload, but my concern is someone is going to think we mean to specify the protocol(s) by listing this item in the scope, and then start talking about VXLAN or the protocol de jour. At the RM, all we care about is that there isolation. Very small instances may even do this VLANs, and not use encapsulation at all (this is what we do in RI today). Hence, there shouldn't be any references to specific encap protocols in the RM, except, possibly as examples.